src/auth/auth.go

```diff
@@ -19,6 +19,11 @@ const (
 	// authenticated context
 	ContextTokenKey = "authn-token"
 
+	// clusterRoleBindingKey is a key in etcd, in the roleBindings collection,
+	// that contains the set of role bindings for the cluster. These are frequently
+	// accessed so we cache them.
+	ClusterRoleBindingKey = "CLUSTER:"
+
```

Comment:
Does this need to be part of a PR specific to projects? I can see that it might, but since it wasn't there earlier ...

Reply:
Basically need to reference this in a lot of different places. Previously it was private in the auth server package, but really it should be here I think.

Reply:
Cool. Should capitalise it in the comment too.